### ISSUE TITLE
feature request: 

### ISSUE THREAD

**Issue by LeonardoMor** [stevearc/conform.nvim#396]
### Did you check existing requests?

- [X] I have searched the existing issues

### Describe the feature

Add [jqfmt](https://github.com/noperator/jqfmt).

### Provide background

Some of keeping our `jq` queries in their own file makes sense. There is an lsp for the `jq` language already and a formatter. It'll be great to have the formatter supported by conform.

### What is the significance of this feature?

nice to have

**Comment by ervinpopescu** [comment]
```lua
      formatters = {
        jqfmt = {
          command = os.getenv("XDG_DATA_HOME") .. "/go/bin/jqfmt",
          args = { "-ob", "-ar", "-op", "pipe" },
        },
    }
```

This config will fail with:
```
21:14:51[ERROR] Formatter 'jqfmt' error: time="2024-07-06T21:14:51+03:00" level=fatal msg="could not read file: open /dev/stdin: no such device or address"
```

Leaving this here for reference.

**Comment by ervinpopescu** [comment]
PR opened upstream: https://github.com/noperator/jqfmt/pull/3

**Comment by LeonardoMor** [comment]
> PR opened upstream: [noperator/jqfmt#3](https://github.com/noperator/jqfmt/pull/3)

Cool. Hopefully it gets merged.

**Comment by noperator** [comment]
Merged upstream PR ✅

**Comment by LeonardoMor** [comment]
Beautiful

**Comment by ervinpopescu** [comment]
@LeonardoMor I don't think we need this issue anymore, should we close it? `conform` doesn't need to support `jq`/`jqfmt` directly, it could just be a custom formatter.

**Comment by LeonardoMor** [comment]
True. But the same can be said about the other formatters

**Comment by ervinpopescu** [comment]
I guess so, @stevearc should make the call then.

**Comment by fira42073** [comment]
@ervinpopescu may I ask you to include the example of how you added a custom formatter? Or update your personal dots repo, I'll take a peek ^-^

Thanks in advance!

**Comment by LeonardoMor** [comment]
Alternatively, we could send a PR. Are there any guidelines to do so? Or to add a formatter to conform so that it is available to config out of the box?

**Comment by LeonardoMor** [comment]
Nevermind. Might not be ready for general use. See [here](https://github.com/noperator/jqfmt/issues/5)

**Comment by ervinpopescu** [comment]
That is indeed one of the things I ran into as well.
